app/controllers/search_controller.rb

```diff
@@ -63,6 +63,17 @@ def chat_channels
     render json: { result: ccm_docs }
   end
 
+  def chat_channels_discoverable
```

Comment:
I still don't think we need this extra search method or this extra search. 

We should be able to execute the search for discoverable channels through the regular chat_channel search endpoint using the necessary params to return the discoverable docs as well as the user viewable channels. Then once the documents are sent to the view, the view can figure out which ones to display where if indeed we want them displayed in different sections. However, from the looks of the PR description, these are just displayed inline with the rest of the channels.

Reply:
We have now used a single chat_channel search endpoint for fetching discoverable channels. Please review and let us know.
Thanks